docs/clusters/eiger.md

```diff
@@ -4,13 +4,11 @@
 Eiger is an Alps cluster that provides compute nodes and file systems designed to meet the needs of CPU-only workloads for the [HPC Platform][ref-platform-hpcp].
 
 !!! under-construction
```

Comment:
How about turning this into a `note` or `info` box, because Eiger.alps is no longer under construction?